test/end-to-end/offer.js

```diff
@@ -1848,6 +1879,37 @@ describe("Offer endpoint tests", () => {
 
         });
 
+        describe("Blocked company", () => {
+
+            let blocked_test_company, test_offer;
+            beforeAll(async () => {
+                blocked_test_company = await Company.create({
+                    name: "blocked test company",
+                    bio: "a bio",
+                    contacts: ["a contact"],
+                    isBlocked: true,
+                    hasFinishedRegistration: true
+                });
+                test_offer = await Offer.create(
+                    generateTestOffer({
+                        owner: blocked_test_company._id,
+                        ownerName: blocked_test_company.name,
+                    })
+                );
+
+                test("should fail to edit offer if company blocked", async () => {
+                    const res = await request()
+                        .post(`/offers/${test_offer.id}`)
```

Comment:
Shuldn't this `send` have be called with `/offers/edit/${test_offer.id} ?

Reply:
Interesting, yes!